test/openbci-sdk-test.js

```diff
@@ -758,22 +586,16 @@ describe('#impedanceTesting', function() {
                 impedanceArray[1].channel.should.be.equal(2);
             });
             describe('#inputP', function() {
-                it('data array present',function() {
-                    impedanceArray[1].P.data.length.should.be.above(0);
-                });
-                it('valid average',function() {
-                    impedanceArray[1].P.should.have.property('average').above(-1);
+                it('got raw impedance value',function() {
```

Comment:
why above -1 and not above 0? I didn't think you could have a negative impedance value